### PR TITLE
Adds more roundstart tips

### PR DESCRIPTION
**Pull request by 81Denton**
This PR adds more roundstart tips. Let me know if you know any good ones as well.

### CODE REVIEW

File: strings/sillytips.txt [hunk 1]
```diff
@@ -28,4 +28,6 @@ Just the tip?
 Some people are unable to read text on a game where half of it is based on text.
 As the Captain, you can use a whetstone to sharpen your fancy fountain pen for extra robustness.
 As the Lawyer, you are the last bastion of roleplay-focused jobs. Even the curator got a whip to go fight people with, that sellout!
-There are at least 11 ways to get through plastic flaps. How many can you name?
\ No newline at end of file
+There are at least 11 ways to get through plastic flaps. How many can you name?
+FEED ME A STRAY CAT
```
Comment (bgobandit) [inline strings/sillytips.txt]: if we're going to add memes can we actually put "never pet a burning dog"/"don't spit into the wind" in

Comment (81Denton) [reply strings/sillytips.txt]: Huh, I must have overlooked that line when squashing commits. Lemme remove tomorrow

Comment (bgobandit) [reply strings/sillytips.txt]: wait never mind this is sillytips

**Comment by Poojawa** [comment]
I feel like a lot of these should be little secrets that people should impart as in-game mentors.

**Comment by 81Denton** [comment]
I'd agree if we had a Goon-style mentor mechanic or if this was a MRP/HRP server where people actually talk to each other.

**Comment by Poojawa** [comment]
Hippie and Citadel both have Mentor systems, and they can be promoted/demoted by anyone with +ADMIN.

And less nullRP, gotta go fast wouldn't hurt. :P Since everything has been balanced to make stuff as fast as possible in every aspect.

**Comment by 81Denton** [comment]
>Hippie and Citadel both have mentor systems

That‘s nice for them, but this is /tg/

**Comment by Poojawa** [comment]
and we're both downstreams of /tg/, silly.

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**